vnext/Microsoft.ReactNative/RedBox.cpp

```diff
@@ -187,14 +185,28 @@ struct RedBox : public std::enable_shared_from_this<RedBox> {
           m_errorStackText.Text(Microsoft::Common::Unicode::Utf8ToUtf16(json["type"].asString()));
         }
         return;
+      } else if (json.count("name") && boost::ends_with(json["name"].asString(), "Error")) {
+        auto message = std::regex_replace(json["message"].asString(), colorsRegex, "");
+        const auto originalStack = std::regex_replace(json["stack"].asString(), colorsRegex, "");
+        std::string stack;
+        if (boost::starts_with(originalStack, json["name"].asString() + ": " + message)) {
```

Comment:
I know this is not executed in a loop, but maybe std::string::append or string streams would be better than plain '+'s